src/System.Linq.Expressions/tests/SequenceTests/SequenceTests.cs

```diff
@@ -2960,29 +2929,6 @@ public static void InvokeQuotedLambda(bool useInterpreter)
             Assert.Equal(6, d());
         }
 
-        [Theory(Skip = "870811")]
```

Comment:
Does anyone know what this 870811 refers to? I'm assuming some MS record of an issue, though the test passes and does so with reasonable speed, so I can't see why it was skipped.